contact.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Contact</title>
+    <link rel="stylesheet" href="homestyle.css">
+  </head>
+  <body>
+  	  <header class="navbar">
+  		  <div id="NC"><a href="home.html">Nick Clifford</a> 
+  		  </div>
+  		  <div class="links">
+  			  	<a id="aboutme" href="aboutme.html">About Me</a>
+  			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+  			 	<a id="contact" href="contact.html">Contact</a>
+  		  </div>
+  	  </header>
+	  
+	  <div class="contactmain">
+		  <div class="contacttitle">How to get in touch
+		  </div>
+		</div>
+		<div class="contactdetails">
+			<div class="contacticons">
+			<div class="mail"><a href="mailto:nickclifford@gmail.com">
+				<img alt="Email" src="mail.png" width="75" height="75">
+				</a>
+			</div>
+			<div class="phone"><a href="tel:07970740492">
+				<img alt="Phone" src="phone.png" width="75" height="75">
```

Comment:
width/height better to set in css

Reply:
Also, recruiters would use this as an invitation to call you night and day. Not a code issue but a heads up!